Tests/UT.Add-AppGatewayKubernetesIngressCert.Tests.ps1

```diff
@@ -0,0 +1,77 @@
+Set-Location $PSScriptRoot\..\Infrastructure-Scripts\
+
+Describe "Add-AppGatewayKubernetesIngressCert unit tests" -Tag "Unit" {
+    Mock Get-AzApplicationGateway -MockWith {
+        return @{
+            Name = "das-foo-ag"
+        }
+    }
+    Mock Get-AzKeyVaultCertificate -MockWith {
+        return @{
+            SecretId = "https://das-dev-shared-kv.vault.azure.net:443/secrets/wildcard-apprenticeships-education-gov-uk/1234567890a1234567b1c1d12efg12h1"
```

Comment:
Does the mock value need to be this specific?

Reply:
The last segment is essential as the script removes this segment so we need to test that it does so correctly.  I've made the earlier segments more general but would like it to be in the same format as the actual data that would be returned